lib/src/executable.dart

```diff
@@ -57,9 +64,53 @@ Future<void> run(List<String> args) async {
   exitCode = await process.exitCode;
 }
 
+Future<void> handleFastFormat(List<String> args) async {
```

Comment:
So the key here in terms of speed is to reconstruct the formatting tool (whichever one it is) from `tool/dart_dev/config.dart` without having to actually load that dart file, right? Because then it can be completely precompiled?

I think we could go a bit further for the non-ORF use case and reconstruct two config options: the formatter to use (dart_style vs dartfmt) and the command line args. If we do that, then all three formatters along with custom line length should be supported.

One idea for the implementation here would be to modify the visitor to reconstruct the format tool (either OverReactFormatTool, FormatTool, or nothing if no `format` entry is found), e.g.:
```dart
Future<void> handleFastFormat(List<String> args) async {
  assertDirIsDartPackage();

  DevTool formatTool;
  final configFile = File(_configPath);
  if (configFile.existsSync()) {
    final configVisitor = ConfigVisitor();
    parseString(content: configFile.readAsStringSync())
        .unit
        .accept(configVisitor);
    formatTool = configVisitor.formatTool; // could be null if no custom `format` entry found
  }
  formatTool ??= chooseDefaultFormatTool();

  try {
    exitCode = await DartDevRunner({'hackFastFormat': formatTool}).run(args);
  } ...
}

DevTool chooseDefaultFormatTool() => ...; // https://github.com/Workiva/dart_dev_workiva/blob/master/lib/src/workiva_config.dart#L34-L46  
```